src/System.IO.FileSystem/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,7 +1,8 @@
 Compat issues with assembly System.IO.FileSystem:
+# These are now virtual in the implementation, which should be ok
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Exists' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Name' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Exists.get()' is abstract in the implementation but is not abstract in the contract.
 CannotMakeMemberAbstract : Member 'System.IO.FileSystemInfo.Name.get()' is abstract in the implementation but is not abstract in the contract.
-TypesMustExist : Type 'System.IO.Enumeration.FileSystemEntry.<_fileNameBuffer>e__FixedBuffer' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.IO.Enumeration.FileSystemEnumerator<TResult>.GetData()' does not exist in the implementation but it does exist in the contract.
+# Not sure why this one is coming up, it is being generated by C# for the Unix version
```

Comment:
Nit. You can change this to the more definitive `# C# generates backing fields for fixed buffers as public` as per the discussion below.